skietbaan_fe/src/components/Login.js

```diff
@@ -23,17 +23,20 @@ class Login extends Component {
     this.setState({
       [target.name]: target.value,
     });
-    let isValid = true;
+    let isValid = false;
     let stateUpdate = {
       invalidPassword: this.state.invalidPassword,
       invalidUsername: this.state.invalidUsername
      }
     if (target.name === "passwordValue" ) {
       stateUpdate.invalidPassword= false;
-    };
+    }
     if (target.name === "usernameValue" ) {
       stateUpdate.invalidUsername= false;
-    };
+      isValid = false;
+    }
+    if(this.state.usernameValue && this.state.passwordValue)
```

Comment:
be consistent with if statements and their use of brackets `{ }`

Reply:
fixed